hs_data_services/hs_data_services/settings.py

```diff
@@ -101,3 +101,48 @@
 USE_L10N = True
 
 USE_TZ = True
+
+
+LOGGING = {
```

Comment:
I couldn't get these logs to show up in `docker compose logs -f` even though they show up in a console when I run manage.py shell... not sure if you see anything that could explain why?